pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/CollectiveDataEdition/CollectiveDataForm/CollectiveDataForm.tsx

```diff
@@ -182,6 +182,7 @@ const CollectiveDataForm = ({
               inline
               maxDisplayOptions={20}
               maxDisplayOptionsLabel="20 résultats maximum. Veuillez affiner votre recherche"
+              hideTags
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Il devrait y avoir un test à update non ?